src/ui/MxcMediaProxy.cpp

```diff
@@ -47,6 +49,11 @@ MxcMediaProxy::MxcMediaProxy(QObject *parent)
                 if (t == QMediaMetaData::Orientation)
                     emit orientationChanged();
             });
+
+    connect(ChatPage::instance()->timelineManager()->rooms(),
+            &RoomlistModel::currentRoomChanged,
+            this,
+            [this] { this->pause(); });
```

Comment:
can you not pass the pause function directly?

Reply:
🤦 Of course I can.

Reply:
Fixed and working